topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -673,7 +673,7 @@ Therefore we offer a parallel tutorial for the 2 methods which give very similar
 
 {% include _includes/cyoa-choices.html option1="featureCounts" option2="STAR" default="featureCounts" %}
```

Comment:
Suggest moving this include line down to just above where the content changes so the learner can see what changes, "Counting reads per genes" section?